### DISCUSSION TITLE
NFC: UID is changing value

### DISCUSSION THREAD

**Discussion by israeloriente** [capawesome-team/capacitor-plugins#200] [Q&A] [answered]
I've been scanning NFC tags and when I use other phones my id result is different. Is there any way to get the tag's unique ID?

**Comment by robingenz** [comment]
The tag ID should always be the same. Do you have a specific example with values? Also, which plugin version do you use?

**Reply by israeloriente** [reply]
I am using "@capawesome-team/capacitor-nfc": "^6.2.0"

**Reply by robingenz** [reply]
Please provide a specific example with values.

**Reply by israeloriente** [reply]
![image](https://github.com/capawesome-team/capacitor-plugins/assets/31784632/cdecf375-cb5b-446c-951c-a6b5be432dbd)

The ID (line 102) changes when I scan from Android and iOS.

**Reply by robingenz** [reply]
This is not a specific example with values.

**Comment by israeloriente** [comment]
It's as if the platform inverted the content.

<p align="center">
  <img src="https://github.com/capawesome-team/capacitor-plugins/assets/31784632/255fc330-c36a-44d8-98a5-71bd49cd259f" alt="iOS" width="300"/>
  <img src="https://github.com/capawesome-team/capacitor-plugins/assets/31784632/b25b0ca4-624f-4400-9c91-70e0b8b7196c" alt="Android (1)" width="300"/>
  <img src="https://github.com/capawesome-team/capacitor-plugins/assets/31784632/bf401b4c-457a-4ecc-a3e9-5f4dea0721f8" alt="iOS" width="300"/>
  <img src="https://github.com/capawesome-team/capacitor-plugins/assets/31784632/5954bbfb-c5de-4531-8189-5e7cc4551cce" alt="Android" width="300"/>
</p>

**Reply by robingenz** [reply]
Which nfc tag do you use? I will try to reproduce this.

**Comment by israeloriente** [comment]
Using other applications the same "error" occurs, is it a bug? In the end we don't have the same UID depending on the platform (information about the tag in the image)

<p align="center">
  <img src="https://github.com/capawesome-team/capacitor-plugins/assets/31784632/70843982-a2c9-47b8-9487-e6c7a9641557" alt="iOS (1)" width="300"/>
  <img src="https://github.com/capawesome-team/capacitor-plugins/assets/31784632/aca3c573-c842-4aa6-b393-5628f7ceab53" alt="Android (2)" width="300"/>
</p>

**Reply by robingenz** [reply]
It's the same UID, it's just reversed. That shouldn't be a problem. I will check if I can reproduce this and create a fix. 

(Please don't create a new thread with every reply.)

**Reply by robingenz** [reply, marked as answer]
I've just published a dev version with a fix: 

```
npm i @capawesome-team/capacitor-nfc@6.2.0-dev.5d268bc.1719086007
```

This fix will be published with the next major release as it could be a breaking change for many.

**Reply by israeloriente** [reply]
Thank you!
